cmd/api/src/api/middleware/middleware.go

```diff
@@ -40,6 +40,11 @@ import (
 	"github.com/unrolled/secure"
 )
 
+const (
+	// Default timeout for any request is thirty seconds
+	defaultTimeout = 30 * time.Second
+)
+
```

Comment:
This and a number of other changes in this commit likely don't need to be reverted: https://github.com/SpecterOps/BloodHound/pull/540/commits/8f3b05baf2ad364de68228b36a57aaf15411781d 
Unsure if we can easily tease these out to isolate this revert to just the code related to the error returns?